tools/one_step_veloxbackend.sh

```diff
@@ -91,7 +86,7 @@ cd $GLUTEN_DIR/ep/build-velox/src
 if [ $BUILD_VELOX_FROM_SOURCE == "ON" ]; then
   ./get_velox.sh
 fi
-./build_velox.sh --build_protobuf=$BUILD_PROTOBUF --build_folly=$BUILD_FOLLY --enable_s3=$ENABLE_S3 \
+./build_velox.sh --build_protobuf=$BUILD_PROTOBUF --enable_s3=$ENABLE_S3 \
```

Comment:
LGTM. An issue want to discuss is, can we compile only one version of Spark in this script at below place? There seems to be some unresolved issue if compiling both.
`
mvn clean package -Pbackends-velox -Pspark-3.2 -Pspark-3.3 -DskipTests
`

Reply:
Thanks for your comments. @PHILO-HE also mention it. 
Changed to 
`mvn clean package -Pbackends-velox -Pspark-3.2 -DskipTests`
`mvn clean package -Pbackends-velox -Pspark-3.3 -DskipTests`

compile it separately.

Reply:
Thanks for the quick fix!